src/main/java/com/vint/shop/service/ManufacturerService.java

```diff
@@ -0,0 +1,15 @@
+package com.vint.shop.service;
+
+import com.vint.shop.domain.Manufacturer;
+
+import java.util.List;
+
+public interface ManufacturerService {
+
+    List<Manufacturer> findAll();
+
+    boolean deleteManufacturer(Long id);
```

Comment:
Same here
If it's a service for the entity, the entity name in the method could be skipped